src/components/Header/header.js

```diff
@@ -8,7 +8,7 @@ import PropTypes from "prop-types";
 import ResponsiveUtils from "../Widgets/ResponsiveUtils/responsiveUtils";
 import ImageOptimize from "../../components/Widgets/ImageOptimize/imageOptimize";
 import NavigationBar from "../Navbar/navbar";
-import {Row, Col } from "reactstrap";
+import {Col} from "reactstrap";
```

Comment:
Multiple imports from reactstrap. Not good

Reply:
fixed